packages/js-dpp/lib/errors/consensus/basic/identity/InvalidIdentityPublicKeySecurityLevelError.js

```diff
@@ -0,0 +1,63 @@
+const AbstractBasicError = require('../AbstractBasicError');
+
+class InvalidIdentityPublicKeyDataError extends AbstractBasicError {
```

Comment:
Public key data sounds like `data` (key itself) field is not valid. Maybe use purpose or security words instead?

Reply:
The name of the file actually already was correct, looks like I've just forgotten to rename the class itself